subsys/mgmt/mcumgr/lib/cmd/os_mgmt/src/os_mgmt.c

```diff
@@ -148,15 +157,33 @@ os_mgmt_taskstat_encode_stack_info(zcbor_state_t *zse,
 #endif /* CONFIG_OS_MGMT_TASKSTAT_STACK_INFO */
 }
 
+static inline bool
+os_mgmt_taskstat_encode_runtime_info(zcbor_state_t *zse, const struct k_thread *thread)
+{
+	bool ok = true;
+
+#ifdef CONFIG_SCHED_THREAD_USAGE
+	k_thread_runtime_stats_t thread_stats;
+
+	k_thread_runtime_stats_get((struct k_thread *)thread, &thread_stats);
+
+	ok = zcbor_tstr_put_lit(zse, "runtime") &&
+	zcbor_uint64_put(zse, thread_stats.execution_cycles);
+#else
+	ok = zcbor_tstr_put_lit(zse, "runtime") &&
+	zcbor_uint32_put(zse, 0);
```

Comment:
Shouldn't this be dependent on `CONFIG_OS_MGMT_TASKSTAT_ONLY_SUPPORTED_STATS`, at all, when `CONFIG_SCHED_THREAD_USAGE` is not enabled this is unsupported stat.

Reply:
Fixed